### PR TITLE
docker compose for production

### PR DESCRIPTION
**Pull request by andnils**
Attempt at docker compose for production use.

Contains a clojure container with copied uberjar.
Contains a nginx container with copied static files.
Vanilla postgres image.

Create file `.env` containing the following config variables:
```
MEASURE_DB=xx
MEASURE_DB_USER=xx
MEASURE_DB_PASSWORD=x
```

Also added a Makefile for building all the stuff.

### CODE REVIEW

**Comment by jherrlin** [comment]
Can we get letsencrypt in here to?

**Comment by jherrlin** [comment]
What about rename this file (https://github.com/andnils/measure/blob/a7c76a0c3bfa2e968207a95356ebc680bee546bb/Dockerfile) to reflect where it belongs? For example Dockerfile.clojure?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8-jdk-alpine
+COPY target/uberjar/measure-0.1.0-SNAPSHOT-standalone.jar /srv/measure.jar
+
+EXPOSE 3000
```
Comment (jherrlin) [inline Dockerfile]: What about rename this file to reflect where it belongs? For example Dockerfile.clojure?

**Comment by andnils** [comment]
Rename `Dockerfile` to `Dockerfile.clojure`? Yup, probably a good idea.
I was also thinking of putting all docker-related files into a separate dir. Maybe keep the dev docker-compose in project root.

**Comment by andnils** [comment]
Also, I'm not really satisfied with all the env-variables...we need a better solution.
We have one set of variables for the postgres-image, one set for the clojure server, then we have the profiles.clj-file, and also the Dockerfile.production.yml...
Disaster!

**Comment by jherrlin** [comment]
Placing all docker-related files in one dir sounds like a good plan!

**Comment by jherrlin** [comment]
Yeah, can we have one source of truth for dev and one for prod? Would make it more simple

**Comment by andnils** [comment]
NOTE TO SELF: move production files (again) to separate dir (prod).
In Makefile: copy jar-file and bundle.js to prod-dir, before building stuff.

**Pull request merged**